jumanji/environments/logic/rubiks_cube/env.py

```diff
@@ -86,7 +85,7 @@ def __init__(
         self,
         time_limit: int = 200,
         reward_fn: Optional[RewardFn] = None,
-        env_viewer: Optional[RubiksCubeViewer] = None,
+        env_viewer: Optional[Viewer[State]] = None,
         generator: Optional[Generator] = None,
```

Comment:
Would it be possible to reorder the init arguments by importance order, i.e. have generator first, then time_limit, reward_fn and viewer.

Reply:
Agreed, similar to other environments